resources/scripts/components/server/Console.tsx

```diff
@@ -201,8 +201,12 @@ export default () => {
         };
     }, [ connected, instance ]);
 
+    const openConsole = () => {
+        window.open(window.location.href + '/console', 'Server Console', 'height=1000,width=1500');
+    };
+
     return (
-        <div css={tw`text-xs font-mono relative`}>
+        <div css={window.location.href.includes('/console') ? tw`text-xs font-mono relative w-full h-screen overflow-hidden` : tw`text-xs font-mono relative`}>
```

Comment:
Can you use the react router helpers to check the route rather than the window location item?